community/item-schema.json

```
@@ -619,22 +625,28 @@
 		{
 			"itemdefid": 100000000,
 			"type": "item",
+			"name_brazilian": "Doador: Lágrimas para Tarnor",
 			"name_english": "Donor: Tears for Tarnor",
 			"name_german": "Spender: Tränen für Tarnor",
 			"name_italian": "Donatore: Lacrime per Tarnor",
+			"name_schinese": "捐赠者: Tears for Tarnor",
+			"description_brazilian": "Representa uma doação ao autor da campanha Lágrimas para Tarnor, Michael 'Ezekel' Abraham.\n\nA renda desta doação vai para o criador de uma campanha. As doações não são necessárias para acessar nenhum conteúdo e não afetam a jogabilidade.",
 			"name_schinese": "捐赠者：塔诺尔之泪",
```

This one got mixed up a little.

```suggestion
			"name_schinese": "捐赠者：塔诺尔之泪",
			"description_brazilian": "Representa uma doação ao autor da campanha Lágrimas para Tarnor, Michael 'Ezekel' Abraham.\n\nA renda desta doação vai para o criador de uma campanha. As doações não são necessárias para acessar nenhum conteúdo e não afetam a jogabilidade.",
```

---

Those were probably caused because the chinese translator edited again after my corrections, tell me in case you need me to correct it again.